src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -165,6 +165,7 @@ private async Task InvokeWorkerAsync(IProgress<CodeAnalysisProgress> progressTra
                 // Clear the progress we showed while computing the action.
                 // We'll now show progress as we apply the action.
                 progressTracker.Report(CodeAnalysisProgress.Clear());
+                progressTracker.Report(CodeAnalysisProgress.Description(EditorFeaturesResources.Applying_changes));
```

Comment:
Now, after cleanup, and selecting "apply" we actually "Applying changes" :)

So user sees 3 passes.

First: Gettins the initial fixes.
Second: Cleaning them. 
Third: Applying them to workspace.